src/server/index.tsx

```diff
@@ -64,7 +65,12 @@ export const createServer = ({
     reportUri: '/__cspreport__',
   };
 
-  const cdnURL = env.PUBLIC_URL || "'none'";
+  const staticSrc = env.PUBLIC_URL
+    ? `${env.PUBLIC_URL}${STATIC_PATH}`
+    : "'none'";
+  const imgSrc = env.PUBLIC_URL
+    ? [staticSrc, `${env.PUBLIC_URL}/favicon.ico`]
```

Comment:
Can't we just put `favicon.ico` inside the `static` directory? That would keep things simple.

Reply:
Yep, that makes sense.